### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ticklemepierce**
Added a few steps to help with the setup of the virtualenv for waterbutler

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,14 @@ invoke server
 
 ### startup commands
 
+In order to run waterbutler, you must have a virtualenv created for waterbutler running python3.3 or higher.  This can be created with the following commands:
```
Comment (sloria) [inline README.md]: Might want to note that the below commands requires `virtualenvwrapper`.

Comment (sloria) [inline README.md]: Also, include the command for activating the virtualenv.

**Comment by lbanner** [comment]
Nice @ticklemepierce

File: README.md [hunk 2]
```diff
@@ -18,13 +18,24 @@ invoke server
...
 ```bash
-# Make sure that you are using >= python3.3
+brew install python3 (installs the latest version of python3)
+which python3 (prints the path to where python3 is installed)
+pip install virtualenv (if you don't already have this)
+pip install virtualenvwrapper (if you don't already have this)
+mkvirtualenv --python=(path to python3) waterbutler
```
Comment (sloria) [inline README.md]: Installing python3 with homebrew will symlink python3 to the correct python3 binary, so you should just be able to do:

```python
mkvirtualenv --python=python3 waterbutler
```

Comment (sloria) [inline README.md]: I'll make this change when I merge

**Pull request merged**